docopt.nimble

```diff
@@ -5,6 +5,7 @@ license = "MIT"
 srcDir = "src"
 
 requires "nim >= 0.15.0"
+requires "regex >= 0.7.4"
 
 task test, "Test":
   exec "nim compile --verbosity:0 --run test/test"
```

Comment:
Change this to ``exec "nimble c --verbosity:0 -r -y test/test`` and you'll be able to get rid of the ``nimble`` lines you've added in the travis file. Repeating what is inside this .nimble file should never be necessary.